django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -505,9 +510,8 @@ JSONEditor.defaults.themes.django = JSONEditor.AbstractTheme.extend({
             if (matchKey) {
                 if (elem[matchKey](selector)) {
                     return elem;
-                } else {
-                    elem = elem.parentNode;
```

Comment:
I don't understand, wouldn't this change also the behaviour of the code?

Reply:
Returning would mean the `elem = elem.parentNode;` part is never reached. The same happens with else. If the condition is matched, run `return elem;`, if not then run `elem = elem.parentNode;` and the rest. With the `return` statement, if the condition is met, the function returns and the rest of the code is never reached. If it does not, it'll continue on executing the rest of the code, including the `elem = elem.parentNode;` part.

There's a good explanation on StackOverflow: https://stackoverflow.com/a/3261899

Reply:
correct, sorry I was going too fast